### ISSUE TITLE
Question: how to server local images?

### ISSUE THREAD

**Issue by kurt-o-sys** [llh911001/slidemd#1]
When I have images stored locally, how to embed them?

```
$ ls
clarity.jpg   presentation.json   presentation.md
```

In my presentation-code:
```
![Slide Image](./clarity.jpg)
```

This seems to fail to find the image. Also `localhost:8080/clarity.jpg` fails, so it seems the presentation is served from another location than the one I start the presentation from and it doesn't copy the other files, assets and so forth.

**Comment by llh911001** [comment]
Hi there, inserting images is a common scenario indeed. 

For me, I'd serve local images through a different port using some local HTTP server, like `python3 -m http.server 9000`, then reference them in the markdown by `![Image](http://localhost:9000/image.png)`.

Hope this helps 😀

**Comment by kurt-o-sys** [comment]
ah, ok, makes sense. thx.

**Comment by llh911001** [comment]
Maybe this is a little late, but I just released v1.1.0 which supports serving local files (including images) in CWD :)
